fail2ban/server/filter.py

```diff
@@ -844,35 +943,50 @@ def getJournalMatch(self, match): # pragma: no cover - Base class, not used
 
 import socket
 import struct
+from .utils import Utils
 
 
 class DNSUtils:
 
 	IP_CRE = re.compile("^(?:\d{1,3}\.){3}\d{1,3}$")
 
+	# todo: make configurable the expired time and max count of cache entries:
+	CACHE_dnsToIp = Utils.Cache(maxCount=1000, maxTime=60*60)
```

Comment:
My point is, why should fail2ban decide the TTL of an NXDOMAIN? Any discussion of what some arbitrary hard-coded TTL in fail2ban should be for DNS answers is beyond my point.

I'm too tired to look up all of the applicable RFCs (maybe @sebres doesn't think we should read RFCs in addition to any PEPs, as previously stated).